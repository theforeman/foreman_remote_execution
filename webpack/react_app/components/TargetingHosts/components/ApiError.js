import React from 'react';
import { translate as __ } from 'foremanReact/common/I18n';

const ApiError = () => (
  <div className="alert alert-danger">
    <span className="pficon pficon-error-circle-o " />
    <span className="text">
      {__(
        'There was an error while updating the status, try refreshing the page.'
      )}
    </span>
  </div>
);

export default ApiError;
