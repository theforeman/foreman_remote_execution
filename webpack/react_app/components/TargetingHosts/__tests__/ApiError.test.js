import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
import ApiError from '../components/ApiError';

const fixtures = {
  'renders ApiError': {},
};

describe('ApiError', () =>
  testComponentSnapshotsWithFixtures(ApiError, fixtures));
