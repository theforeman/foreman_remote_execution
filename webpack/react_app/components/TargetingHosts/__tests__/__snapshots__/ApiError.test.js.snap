// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApiError renders ApiError 1`] = `
<div
  className="alert alert-danger"
>
  <span
    className="pficon pficon-error-circle-o "
  />
  <span
    className="text"
  >
    There was an error while updating the status, try refreshing the page.
  </span>
</div>
`;
